test/squeeze.lean

```diff
@@ -17,7 +17,7 @@ meta def squeeze_simp_test
 do (cfg',c) ← parse_config cfg,
    squeeze_simp_core slow_and_accurate.is_some no_dflt hs
      (λ l_no_dft l_args, simp use_iota_eqn none l_no_dft l_args attr_names locat cfg')
-     (λ args, guard ((args.map to_string).perm (l.map to_string)) <|> fail!"{args} expected.")
+     (λ a, guard (a.map to_string = l.map to_string) <|> fail!"{a.map to_string} expected.")
```

Comment:
@arthurpaulino, as I understand, `simp` needs the arguments to be in order, right? so I think removing this `perm` from the check and replacing it with equality will ensure your example won't break

Reply:
Right, good catch!